docs/self-managed/components/orchestration-cluster/identity/overview.md

```diff
@@ -9,9 +10,9 @@ import TabItem from '@theme/TabItem';
 
 Identity is included by default with any [orchestration cluster](/self-managed/reference-architecture/reference-architecture.md#orchestration-cluster). Within an Orchestration Cluster, Identity provides unified, cluster-level identity management and authorization.
 
-Identity is available via [Helm install](/self-managed/installation-methods/helm/install.md) and for local development via [Camunda 8 Run](/self-managed/quickstart/developer-quickstart/c8run.md).
+Identity is available via [Helm install](/self-managed/installation-methods/helm/install.md) and for local development via [Camunda 8 Run](/self-managed/quickstart/developer-quickstart/c8run.md) and [Docker Compose](/self-managed/quickstart/developer-quickstart/docker-compose.md).
```

Comment:
We do we list only Helm install here? With 8.8 we should have Docker, Helm and manual install available